odoo/addons/base/models/ir_asset.py

```diff
@@ -36,14 +33,59 @@ def fs2web(path):
         return path
     return '/'.join(path.split(os.path.sep))
 
+
 def can_aggregate(url):
     parsed = urls.url_parse(url)
     return not parsed.scheme and not parsed.netloc and not url.startswith('/web/content')
 
+
 def is_wildcard_glob(path):
     """Determine whether a path is a wildcarded glob eg: "/web/file[14].*"
     or a genuine single file path "/web/myfile.scss"""
-    return not WILDCARD_CHARACTERS.isdisjoint(path)
+    return '*' in path or '[' in path or ']' in path or '?' in path
+
+
+def _glob_static_file(pattern):
+    # simplified glob support (fnmatch is not precise enough)
```

Comment:
[`pathlib.Path.glob`](https://docs.python.org/3.7/library/pathlib.html#pathlib.Path.glob) handle `**`, no need to reimplement it ourself.

Reply:
I'll need to make this test again to give exact values.